src/components/common/fetchData.js

```diff
@@ -0,0 +1,88 @@
+export async function getContentList(contentCategory, pageID = 1, tag_id="", search="") {
```

Comment:
@VortexExpansion 
`tag_id="",` seems not processed with Prettier, pls check your formatter config?

Reply:
@yabe-diverta  san

Does not show up on my prettier formatter. But changed default value to `tag_id = null` instead. 

Can you confirm now ?

Reply:
Hmm reconsidered, it would be better to set up the config in PJ 🤔 
how do you think?

Reply:
@yabe-diverta san

I added a template .`prettierrc` file https://github.com/diverta/front_next_media/pull/5/commits/0e4f78b341f4fd8ab37053d1410b8390835c768b

Please feel free to add any of the content you think is important and missing. 

What do you think about this ?

Reply:
Yes it's sufficient.
closing this conversation.